### DISCUSSION TITLE
State Management approach

### DISCUSSION THREAD

**Discussion by mboyamike** [droidconKE/flutterconKEApp#12] [Polls]
The topic everyone likes to argue about

Curious on the preferred approach

**Comment by dev-vickie** [comment]
I think bloc does the separation well,so that other guys who aren't much familiar with it can still work on the app

**Comment by ChristineWasike** [comment]
Uuw! Good question. I'm happy to contribute. I'm unfamiliar with Bloc, but I'm happy to go with it.
@mboyamike, @dev-vickie - will you guys be setting it up?

**Reply by mboyamike** [reply]
Oh they have great documentation - https://bloclibrary.dev/

Curious, which one are you conversant with? I am more conversant with riverpod but I think both would work well.

Yeah, I'll help set it up in a bit. Was currently working on a bit of documentation and how to deal with data fetching

**Reply by ChristineWasike** [reply]
I've been using riverpod.
Thank you for the docs!
